.github/workflows/specs.yml

```diff
@@ -11,6 +11,20 @@ jobs:
         os:
           - ubuntu-latest
           - macos-13
+        test:
+          - "tests/fast/*"
+          # For some reason, after running these tests,
+          # `source ~/.rvm/scripts/rvm` fails on macOS, so run them last.
+          # See https://github.com/rvm/rvm/pull/5387#issuecomment-2009391015
+          # These tests also change the default ruby, which is another reason to run them last.
```

Comment:
This comment applies to `tests/fast/*`, not `tests/long/truffleruby_comment_test.sh`.
But I think best to remove it, it's not relevant anymore since they run in separate jobs.

Reply:
Right, I missed that. Well caught.